example/advection/advection_package.cpp

```diff
@@ -167,6 +167,10 @@ std::shared_ptr<StateDescriptor> Initialize(ParameterInput *pin) {
                  std::vector<int>({vec_size}), advected_labels);
     pkg->AddField(field_name, m);
   }
+  // TODO(LFR): Remove this
+  Metadata m_fine(
+      {Metadata::Cell, Metadata::Independent, Metadata::Fine, Metadata::FillGhost});
+  pkg->AddField("advected_fine", m_fine);
```

Comment:
Not sure we should remove this actually... Unless we add a separate test somewhere?

Reply:
I just thought we should maybe take it out since there is a test called advection performance and this probably has some impact on performance. Not sure if anyone uses that as an actual performance test though... 

Looking at this makes me want to re-write the example though.

Reply:
Yeah that's fair. It's frustrating that we don't have a way to test things like this outside of regression testing. And yes agreed. It needs a rewrite.

Reply:
leaving this in here now I think